bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/tcc/GuideConfig.java

```diff
@@ -0,0 +1,177 @@
+package edu.gemini.wdba.tcc;
+
+import edu.gemini.shared.util.immutable.*;
+import edu.gemini.spModel.gemini.altair.AltairAowfsGuider;
+import edu.gemini.spModel.gemini.altair.AltairParams;
+import edu.gemini.spModel.gemini.altair.InstAltair;
+import edu.gemini.spModel.gemini.gems.Canopus;
+import edu.gemini.spModel.gemini.gmos.GmosOiwfsGuideProbe;
+import edu.gemini.spModel.gemini.gsaoi.Gsaoi;
+import edu.gemini.spModel.gemini.gsaoi.GsaoiOdgw;
+import edu.gemini.spModel.guide.GuideProbe;
+import edu.gemini.spModel.obscomp.SPInstObsComp;
+import static edu.gemini.spModel.target.obsComp.PwfsGuideProbe.*;
+import edu.gemini.wdba.glue.api.WdbaGlueException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ *  Class to evaluate the {@link ObservationEnvironment} and produce a guide config and guide config  name.
+ */
+public final class GuideConfig extends ParamSet {
```

Comment:
__Disclaimer__: This looks like all new code because I renamed it (from `GuideConfigSouth`).  I did modify this class but for the most part it is the same as the original `GuideConfigSouth`.